metadata_service/proxy/gremlin_proxy.py

```diff
@@ -1040,9 +1042,7 @@ def get_table(self, *, table_uri: str,
         readers = self._get_table_readers(table_uri=table_uri)
 
         users_by_type: Dict[str, List[User]] = {}
-        users_by_type['owner'] = sorted(
-            _safe_get_list(result, f'all_owners', transform=self._convert_to_user) or [],
-            key=attrgetter('user_id'))
+        users_by_type['owner'] = _safe_get_list(result, f'all_owners', transform=self._convert_to_user) or []
```

Comment:
User_id was not always defined. I can switch the sort to be on another attribute if you all prefer.